nautobot/docs/user-guides/relationships.md

```diff
@@ -0,0 +1,108 @@
+# Relationships
+
+Relationships are defined between one (or more) objects in your source of truth to reflect business logic or other relationships that may be useful to you but that haven't been defined. Relationships define specific links between objects that might be specific to your network or data.
+
+For more technical details on how to use this feature, please see the documentation on [Relationships](../models/extras/relationship.md).
+
+## Relationship details
+
+This table defines repository parameters that are required to establish a relationship.
+
+|Field|Explanation|
+|:--|:--|
+|Name|Name of the relationship as displayed to users.|
+|Slug|Internal name of this relationship. Please use underscores rather than dashes.|
+|Description|(Optional) Meaningful description to provide additional relationship details.|
+|[Type](../models/extras/relationship.md#relationship-types)|Cardinality of this relationship.|
+|Source type|The source object type to which this relationship applies.|
+|Source Label|(Optional) Label for related destination objects, as displayed on the source object.|
+|Source filter|(Optional) Filterset filter matching the applicable source objects of the selected type.|
+|Destination type|The destination object type to which this relationship applies.|
+|Destination Label|(Optional) Label for related destination objects, as displayed on the destination object.|
+|Destination filter|(Optional) Filterset filter matching the applicable destination objects of the selected type.|
+
+As seen in the table above relationships offers many additional optional options for more advanced functionality, those options will be discussed further during this user guide.
+
+## Using a Relationship
+
+This section will focus on examples relationships.
+
+### Adding a Relationship
+
+To create a relationship, navigate to **Extensibility > Data Management > Relationships**
+
+A simple relationship between `Device to Vlan` which is a `many-to-many` relationship. This relationship is `many-to-many` because more than one Devices can be assigned more than one Vlans. This relationship object is shown below.
+
+![many-to-many example](./images/relationships/01-relationships.png)
```

Comment:
I'm on a bit of a quest lately to make our relationship examples more "realistic". Part of that, IMHO, is that we should almost always be specifying an appropriate `Source Label` and `Destination Label` to provide context around what the meaning of the relationship is. Having a Device view show (for example) "Configured VLANs: 3 VLANs" is a lot more meaningful than the default of "VLANs: 3 VLANs", and likewise having the VLAN view show (for example) "In use on devices: 7 devices" is better than "Devices: 7 devices".

I see that you've addressed this a bit in the next section but think it'd be good to emphasize even more.

Reply:
Thanks, taking a look at some options.

Reply:
Added some context around your suggestion @glennmatthews think this should cover it.  Happy to go into more examples if needed.